src/core/src/pass/serialize.cpp

```diff
@@ -812,6 +911,9 @@ void serialize_rt_info(pugi::xml_node& root, const std::string& name, const ov::
     }
 }
 
+// TODO: Make this definitiion unuque, now it is duplicated between this file and compress_float_constants.cpp
+const std::string& postponed_fp16_compression_tag = "postponed_fp16_compression";
```

Comment:
you can use `extern` here to _declare_, so the definition will be done only in that place.

Reply:
Addressed this issue here. Even better than using `extern`is to use rt_info class defined in `transformations/rt_info`